src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/AbstractAppSchemaTestSupport.java

```diff
@@ -607,13 +607,12 @@ protected NestedFilterToSQL createNestedFilterEncoder(FeatureTypeMapping mapping
     }
 
     /**
-     * Returns xml String from Document Object
+     * Utility method that converts a XML document object to a string.
      *
-     * @param document
-     * @return
-     * @throws TransformerException
+     * @param document Xml Document to parse
+     * @return String representation of xml document
      */
-    protected String toString(Document document) throws TransformerException {
+    protected static String toString(Document document) throws TransformerException {
```

Comment:
Why non ``static`` anymore ?

Reply:
For use the Document to String convert logic without repeating code on other parts of testing code.